manifests/init.pp

```diff
@@ -15,6 +15,8 @@
 #
 # $proxy_password::     Proxy password for authentication
 #
+# $max_speed::          maximum bandwidth used per download in bytes/sec
```

Comment:
To me this was a bit ambiguous. Maybe ```pulp_max_speed``` would be clearer? Should this also be an ```Optional[Integer]```?